Builds/CMake/CMakeFuncs.cmake

```diff
@@ -268,8 +268,21 @@ endmacro()
 ############################################################
 
 # Params: Boost components to search for.
-macro(find_boost)
-  if (NOT WIN32)
+macro(get_boost)
```

Comment:
Nit: I prefer `use` to `get`. These macros are actually changing compiler search paths. `use_boost` sounds like it's going to do something. Both `find_boost` and `get_boost` sound like pure functions.